.gitignore

```diff
@@ -1,104 +1,35 @@
-# Logs
-logs
+# directories
+.yarn/*
+!.yarn/patches
+!.yarn/releases
+!.yarn/plugins
+!.yarn/sdks
+!.yarn/versions
+**/artifacts
+**/build
+**/cache
+**/coverage
+**/.coverage_artifacts
+**/.coverage_cache
+**/.coverage_contracts
+**/dist
+**/node_modules
+
+# files
+*.env
 *.log
+.pnp.*
+coverage.json
 npm-debug.log*
```

Comment:
Everything that was added looks great to me. I'm not sure about this npm-debug.log file, since we're using yarn maybe we can delate it but is something that is not from this commitment.